test/rubygems/test_gem_package_task.rb

```diff
@@ -1,22 +1,27 @@
 # frozen_string_literal: true
 require 'rubygems/test_case'
 require 'rubygems'
-require 'bundler/errors'
-begin
-  require 'rubygems/package_task'
-rescue LoadError, Bundler::GemfileNotFound
-end
+require 'rubygems/package_task'
 
 class TestGemPackageTask < Gem::TestCase
 
   def setup
     super
 
     Rake.application = Rake::Application.new
-    RakeFileUtils.verbose_flag = false
+
+    @prev_rake_fileutils_verbosity = RakeFileUtils.verbose_flag
```

Comment:
I personally feel like "original" says more than "prev" but is up to you

Reply:
Agreed, will update!

Reply:
Updated the PR @bronzdoc, thanks for your review!